src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree.html

```diff
@@ -34,9 +43,17 @@ <h5>
                     {{group.name}}
                 </a>
             </h5>
-            <button data-element="tree-item-options" class="umb-options umb-outline btn-reset sr-only sr-only--focusable sr-only--hoverable" ng-hide="group.isContainer || !group.menuUrl" ng-click="options(group, $event)" ng-swipe-right="options(group, $event)">
-                <i></i><i></i><i></i>
-            </button>
+
+            <!-- TODO: Fix "Open context node" as a label-key or something that's passed through the umb-tree controller -->
```

Comment:
What does this mean?

Reply:
It's just a note about the "Open content node for" text needing to be added as a dictionary key.... I noticed the translation was handled within the controller and I thought it would be nice to od it in the same fashion with this label at a later point. I can remove the comment if you would like? 😃

Reply:
I removed the comment.